views/system.ejs

```diff
@@ -26,6 +26,12 @@
                                             <%= systemInformation.warnings.message %>
                                         </div>
                                     <% } %>
+                                    <% if(serviceInformation.isPm2 !== 'undefined' && !serviceInformation.isPm2){ %>
+                                            <div class="alert alert-warning" role="alert">
```

Comment:
Smoke-test with supertest for containing 'Your system is not running with PM2'.
https://www.codota.com/code/javascript/functions/supertest/Response/text

PM2 can easily be mocked by overriding (bit dirty yeah) the environment variable which is checked in `isPm2()`

Reply:
Check for the PID inside the isPm2 function maybe? would be much harder to override and can be then backfilled to the restart function.

Reply:
I dont see why actually. What would be your reasoning for making `isPm2()` checking for PID?
Btw, I edited my comment above with the way on how to mock the env variable

Reply:
The only reasoning behind that is it can't be mocked. It has to be running as a pm2 service to get back a response from pm2 itself. It's a certain, this is pm2 to me.